Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-alpine3.18
+LABEL maintainer="roman"
+
+ENV PYTHOUNNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py", "runserver", "0.0.0.0:8000"]
```

Comment:
Ensure that the path `app/main.py` is correct and that the `runserver` command is appropriate for your application. If you are using Django, the command should be `python manage.py runserver 0.0.0.0:8000`.